src/styles/watch.scss

```
@@ -0,0 +1,63 @@
+
+.watch {
+  position: relative;
+  margin: 100px auto;
+  width: 600px;
+  height: 600px;
+
+  border: 10px solid cadetblue;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+
+  &__center {
+    position: absolute;
+    bottom: 47%;
```

You can use transform translate and leave here 50%

```suggestion
    bottom: 50%;
```